src/main/java/com/start/code/BubbleSort.java

```diff
@@ -0,0 +1,27 @@
+package com.start.code;
+
+public class BubbleSort {
```

Comment:
Add methods to sort other number types (i.e. double, long), and String array and a List.